### PR TITLE
Update Manual Testing for DG

### PR DESCRIPTION
**Pull request by mrmrinal**
Update the Manual Testing cases for DG

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -1013,10 +1013,71 @@ testers are expected to do more *exploratory* testing.
       than the number of assets/liabilities/policies listed).
       Expected: Similar to previous.
 
+### 7.6 Clearing client list
+
+1. Clearing all Clients in the home window.
+    1. Prerequisites: currently in the home window and there is at least 1 client.
```
Comment (Bluntsord) [inline docs/DeveloperGuide.md:1019]: Client is currrently in the home window pls. or at least make ur c C

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -1013,10 +1013,71 @@ testers are expected to do more *exploratory* testing.
...
+    2. Test case: `clear` <br>
+       Expected: All the clients listed in the client list panel will be removed, and the client list panel will be cleared. A
```
Comment (Bluntsord) [inline docs/DeveloperGuide.md:1021]: `ClientListPanel`

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -1013,10 +1013,71 @@ testers are expected to do more *exploratory* testing.
...
+       message confirming that 'DonnaFin has been cleared!' will appear in the result display.
```
Comment (Bluntsord) [inline docs/DeveloperGuide.md:1022]: Do they really need the ' here

Comment (Bluntsord) [reply docs/DeveloperGuide.md:1022]: `ResultDisplay`_

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -1013,10 +1013,71 @@ testers are expected to do more *exploratory* testing.
...
+    3. Test case: `clear 2` <br>
+       Expected: Clients will not be cleared from the client list panel. A message saying 'Invalid command format! Try using the help command.'
+       will be shown in the results display.
+
+3. Clearing all Clients in the client window.
+   1. Prerequisites: currently in the client window in any tab.
```
Comment (Bluntsord) [inline docs/DeveloperGuide.md:1028]: Same issue

File: docs/DeveloperGuide.md [hunk 5]
```diff
@@ -1013,10 +1013,71 @@ testers are expected to do more *exploratory* testing.
       than the number of assets/liabilities/policies listed).
       Expected: Similar to previous.
 
+### 7.6 Clearing client list
+
+1. Clearing all Clients in the home window.
```
Comment (sheehui) [inline docs/DeveloperGuide.md:1018]: think ur clients should be in lower case for all headers

File: docs/DeveloperGuide.md [hunk 6]
```diff
@@ -1013,10 +1013,71 @@ testers are expected to do more *exploratory* testing.
...
+
+3. Clearing all Clients in the client window.
+   1. Prerequisites: currently in the client window in any tab.
+   2. Test case `clear` <br>
+      Expected: Clients will not be cleared from the client list panel. A message saying 'Unknown Command' will appear
+      in the result display.
+
+### 7.7 Switching tab in Client Window
+
+1. Switching to Contact Tab
```
Comment (Bluntsord) [inline docs/DeveloperGuide.md:1035]: tab

**Review by sheehui — COMMENTED** [review]
good job. for consistency sake, @DonnyS57 should add the actual error msg to his manual testings as well

**Review by Bluntsord — CHANGES_REQUESTED** [review]
Thanks for great work. Some small comments here and there but overall great job!

File: docs/DeveloperGuide.md [hunk 7]
```diff
@@ -1013,10 +1013,71 @@ testers are expected to do more *exploratory* testing.
...
+       'Input command is invalid: 'NOTIES' does not match any tab' will be shown in the result display.
+
 ### 7.6 Saving data
 
 1. Dealing with missing/corrupted data files
 
-   1. _{explain how to simulate a missing/corrupted file, and the expected behavior}_
+   1. Prerequisites: DonnaFin is closed and there is access to the `[JAR file location]/data/donnafin.json` file.
+      1. Test case: Replace the name, phone, email or address attribute to `null` in the `[JAR file location]/data/donnafin.json` file.
+      2. Expected: When DonnaFin is opened the client list panel will be empty. Entering any valid command after this state will wipe the
```
Comment (sheehui) [inline docs/DeveloperGuide.md:1081]: comma after opened

File: docs/DeveloperGuide.md (hunk 3 above)
Comment (sheehui) [reply docs/DeveloperGuide.md:1022]: don't think need code fence

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (sheehui) [reply docs/DeveloperGuide.md:1021]: don't think need this eh

File: docs/DeveloperGuide.md (hunk 5 above)
Comment (bharathcs) [reply docs/DeveloperGuide.md:1018]:
```suggestion
1. Clearing all clients in the home window.
```

Comment (bharathcs) [reply docs/DeveloperGuide.md]: done

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (bharathcs) [reply docs/DeveloperGuide.md]: done

File: docs/DeveloperGuide.md (hunk 6 above)
Comment (bharathcs) [reply docs/DeveloperGuide.md:1035]: not needed i think

File: docs/DeveloperGuide.md (hunk 7 above)
Comment (bharathcs) [reply docs/DeveloperGuide.md]: done

**Review by Bluntsord — APPROVED** [review]

**Pull request merged**